types/taiko/index.d.ts

```diff
@@ -56,6 +56,7 @@ export interface ReloadOptions extends NavigationOptions {
 export interface ClickOptions extends NavigationOptions, ForceOption {
   button?: 'left' | 'right' | 'middle';
   clickCount?: number;
+  position: string;
```

Comment:
Shouldn't this be 
```suggestion
  position?: string;
```

As position is optional?